docs/tempo/website/getting-started/traces.md

```diff
@@ -0,0 +1,67 @@
+---
+aliases:
+- /docs/tempo/getting-started/traces
+description: "What are traces?"
+keywords:
+- Grafana
+- traces
+- tracing
+title: What are traces?
+weight: 120
+---
+
+# What are traces?
+
+A trace represents the whole journey of a request or an action as it moves through all the nodes of a distributed system, especially containerized applications or microservices architectures. Traces let you profile and observe systems, making it easy to discover bottlenecks and interconnection issues.
+
+## Example of traces
+
+A user on your website enters their email address into a form to sign up for your mailing list. They click **Enter**. This initial transaction has a trace ID which is subsequently associated with every interaction in the chain of processes within a system.
+
+The user's email address is data that flows through your system. In a cloud computing world, it is possible that clicking that one button causes data to touch multiple nodes across your cluster of microservices.
+
+The email address may be sent to a verification algorithm sitting in a microservice that exists solely for that purpose. If it passes the check, the information is stored in a database.
+
+Along the way, an anonymization node strips personally identifying data from the address and sends metadata collected to a marketing qualifying algorithm to determine whether the request was sent from a targeted part of the internet.
+
+Services respond and data flows back from each, sometimes triggering new events across the system. Along the way, logs are written in various nodes with a time stamp showing when the info passed through.
+
+Finally, the request and response activity ends and a record of that request is sent to Grafana or Grafana Cloud.
+
+## Traces and labels
+
+Setting up tracing adds a label to all of these events. The label is generated when the request is initiated and that same label is applied to every single event as the request and response generate activity across the system.
+
+That label enables one to trace, or follow a request as it flows from node to node, service to microservice to lambda function to wherever it goes in your chaotic, cloud computing system and back again. This is recorded and displayed as spans. Each span is the amount of time between a request and response from a specific service in the request's journey through your system.
```

Comment:
Should we italicize the first instances of trace and span here to indicate their special meaning?

Reply:
> Each span is the amount of time between a request and response from a specific service in the request's journey through your system.

This definition conflicts with the one in the terminology list. 

> Represents a specific service or node of the full trace (payment service, inventory, etc.)

In a conversation with some k6 developers, they suggested that a span was a node. And Span could have smaller spans. And that the whole tree made up a trace.

This is the definition I see on most observability tools on Google, (e.g. splunk), but I've also seen "a unit of work" (openTelemetry) and "the amount of time in a context" (tracing.rs).

I think this is terminology that should be nailed down with some developers. In effect it's not that different--either a span is a node, and the trace measures the time between spans; or a span is the time between nodes, and a trace measures the node journey and time of all spans.

But, being totally clear about these terms could help readers.

Reply:
Thank you for the clear and concise write-up of the definitions you've seen. 

I'll ask the Tempo team what definition they would consider to be the best definition for span and for trace.

Reply:
so far, the favorite definition is derived from the OTEL one: A span represents a unit of work done within a trace. 

@hedss has also written a longer definition that we could use in the text: 

> Traces are composed of one or more spans. A span is a unit of work within a trace that has a start time relative to the beginning of the trace, a duration, and an operation name for the unit of work. It usually has a reference to a parent span (unless it is the first span in a trace). It may optionally include key/value attributes that are relevant to the span itself, for example the HTTP method being used by the span, as well as other metadata such as sub-span events or links to other spans.